sleap/gui/commands.py

```diff
@@ -3018,8 +3026,29 @@ def set_visible_nodes(
                 # We don't want to copy a PredictedPoint or score attribute.
                 x_old = copy_instance[node].x
                 y_old = copy_instance[node].y
-                x_new = x_old * scale_width
-                y_new = y_old * scale_height
+
+                # Handles if the new instance is created with right click
+                if (
+                    location is not None
+                    and offset_x is not None
+                    and offset_y is not None
```

Comment:
Do we ever expect the offsets to be `None`? I don't believe we initialize either `offset_x` or `offset_y` and they are only ever defined if `location is not None` (in which case I expect neither `offset_x` nor `offset_y` to be `None`)